### PR TITLE
Rename and update README

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,62 @@
 # JupiterOne Repository Relationship Generator
 
-A script that generates JupiterOne entity relationships between a repository and its dependencies.
-This script will look for dependencies within a repository's `package.json` file as well as the `dependencies.yaml`file if the repository has a `deploy` directory.
+A script that generates JupiterOne entity relationships between a repository and
+its dependencies. This script will look for dependencies within a repository's
+`package.json` file as well as the `dependencies.yaml`file if the repository has
```
Comment (isaacjwilliams) [inline README.md]:
```suggestion
`package.json` file as well as the `dependencies.yaml` file if the repository has
```

File: README.md [hunk 2]
```diff
@@ -1,39 +1,62 @@
...
+
+This script currently works with node.js dependencies in package.json only. It
+does not scan for devDependencies.
 
 ## Before Running
-- Create a directory containing the repositories that you would like the script to be run on (Note: You will need the path to this directory while running the script).
 
-- Ensure that the `package.json` for each repository is at the root of the repository.
+- Create a directory containing the repositories that you would like the script
+  to be run on (Note: You will need the path to this directory while running the
```
Comment (isaacjwilliams) [inline README.md]:
```suggestion
  to be run against (Note: You will need the path to this directory while running the
```

File: README.md [hunk 3]
```diff
@@ -1,39 +1,62 @@
...
 
-A script that generates JupiterOne entity relationships between a repository and its dependencies.
-This script will look for dependencies within a repository's `package.json` file as well as the `dependencies.yaml`file if the repository has a `deploy` directory.
+A script that generates JupiterOne entity relationships between a repository and
+its dependencies. This script will look for dependencies within a repository's
+`package.json` file as well as the `dependencies.yaml`file if the repository has
+a `deploy` directory.
+
+This script currently works with node.js dependencies in package.json only. It
+does not scan for devDependencies.
```
Comment (isaacjwilliams) [inline README.md]: This phrasing is a little confusing... Maybe something like "This script only creates relationships for `dependencies` (not `devDependencies`)". NBD though if you don't want to make another commit.

I'd add this comment as a suggestion so you could just adopt it if you wanted, but GitHub doesn't support multiline suggestions. :(

**Review by isaacjwilliams — APPROVED** [review]
`camelCase` file names (as opposed to `kebab-case`) are more idiomatic for Typescript (and Javascript), but nbd if you don't want to change them all again. :laughing:

**Comment by erkangz** [comment]
> `camelCase` file names (as opposed to `kebab-case`) are more idiomatic for Typescript (and Javascript), but nbd if you don't want to change them all again. 😆

TIL.  All other repos we have currently use `kebab-case`, so... should we change?

**Comment by isaacjwilliams** [comment]
@erkangz Looks like I was wrong.... I got Typescript conventions confused with Javascript conventions since I don't often use bare Javascript.

`camelCase` should be used for Typescript: https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#filename

`kebab-case` should be used for Javascript: https://google.github.io/styleguide/jsguide.html#file-name

Sorry! :sweat:

**Review by isaacjwilliams — APPROVED** [review]
:heart: the refactors to get-repos, more readable now.
